tests/sles4sap/qesapdeployment/test_cluster.pm

```diff
@@ -19,26 +19,41 @@ sub run {
     my $chdir = qesap_get_terraform_dir(provider => $provider_setting);
     assert_script_run("terraform -chdir=$chdir output");
     my @remote_cmd = (
-        'pwd',
-        'uname -a',
+        'pwd', 'uname -a',
         'cat /etc/os-release',
         'sudo SUSEConnect --status-text',
-        'zypper ref',
-        'zypper lr',
-        'zypper in -f -y vim'
+        'zypper ref', 'zypper lr',
+        'zypper in -f -y vim',
+        'zypper -n in ClusterTools2'
```

Comment:
> Install 'ClusterTools2' as 'cs_wait_for_idle' needs it.

Maybe this one can become a comment in the code

Reply:
why `vim` is installed with `-f -y` and `ClusterTools2` only with `-n` ?

Reply:
I only added 'zypper -n in ClusterTools2'. I used -n as "-n means Select packages by their name, don’t try to select by capabilities."

Reply:
> > Install 'ClusterTools2' as 'cs_wait_for_idle' needs it.
> 
> Maybe this one can become a comment in the code

Good idea.